mongo/src/main/java/org/axonframework/extensions/mongo/serialization/DBObjectToStringContentTypeConverter.java

```diff
@@ -41,6 +41,7 @@
 
     @Override
     public String convert(DBObject original) {
-        return JSON.serialize(original);
+        // TODO assuming this is BasicDbObject always
```

Comment:
[This](https://mongodb.github.io/mongo-java-driver/3.6/javadoc/index.html?com/mongodb/util/JSON.html) API page states the `JSON` utility was replaced by the `BasicDBObject` toJson/parse methods. Hence your assumption is right and as such you can drop this line of commentary.

Reply:
This TODO can still be removed.